groups.go

```diff
@@ -384,6 +385,7 @@ type CreateGroupOptions struct {
 	SharedRunnersMinutesLimit       *int                                    `url:"shared_runners_minutes_limit,omitempty" json:"shared_runners_minutes_limit,omitempty"`
 	ExtraSharedRunnersMinutesLimit  *int                                    `url:"extra_shared_runners_minutes_limit,omitempty" json:"extra_shared_runners_minutes_limit,omitempty"`
 	IPRestrictionRanges             *string                                 `url:"ip_restriction_ranges,omitempty" json:"ip_restriction_ranges,omitempty"`
+	AllowedEmailDomainsList         *string                                 `url:"allowed_email_domains_list,omitempty" json:"allowed_email_domains_list,omitempty"`
```

Comment:
@jeremad is this really a supported field for the create endpoint? It's at least not documented, see https://docs.gitlab.com/ee/api/groups.html#create-a-group.

Reply:
No you are right, it is only available for updates, I copied the pattern of `ip_restriction_ranges` which works the same way. But this may be bugged as well. I'll dig into that :pick: